packages/affiliates/test/mocha/affiliates.js

```diff
@@ -21,129 +32,249 @@ const devRewardsToDistribute = "50000";
 // mocks
 const { Queries, Coingecko, SynthPrices } = mocks;
 
-const { getWeb3 } = require("@uma/common");
-const web3 = getWeb3();
-const { toWei } = web3.utils;
-
 describe("DeployerRewards", function() {
-  let affiliates;
-  before(function() {
-    const queries = Queries(datasetPath);
-    const coingecko = Coingecko(datasetPath);
-    const synthPrices = SynthPrices(datasetPath);
-    affiliates = DeployerRewards({
-      queries,
-      empAbi,
-      empCreatorAbi,
-      coingecko,
-      synthPrices
+  describe("CalculateRewards Simple Data", function() {
```

Comment:
this is the new section of tests that were added. The rest of the diffs were from merge issues and shouldnt have changed, though its hard to see.